Applications/Browser/Tab.cpp

```diff
@@ -231,6 +232,19 @@ Tab::Tab()
         GUI::Application::the().quit();
     }));
 
+    auto& view_menu = m_menubar->add_menu("View");
+    view_menu.add_action(GUI::CommonActions::make_fullscreen_action(
+        [&](auto&) {
```

Comment:
It's a dangerous habit to capture using `[&]` when creating a lambda that will persist (such as those we attach as callbacks to a `GUI::Action`), since it captures all stack variables in the surrounding context by reference, and these references will become invalid once we leave the `Tab` constructor!

Instead, just capture `[this]` which gives you access to everything you need anyway. :)